lib/PHPCfg/Parser.php

```diff
@@ -1010,9 +1014,16 @@ protected function parseExpr_Isset(Expr\Isset_ $expr) {
         );
     }
 
+	protected function parseExpr_List(Expr\List_ $expr) {
```

Comment:
This should not be necessary, as list() assignments are handled by parseListAssignment().